charts/openstack-cluster/ci/kube-1-26-0-ha-values.yaml

```diff
@@ -0,0 +1,12 @@
+---
+kubernetesVersion: 1.26.0
+
+machineImageId: 5eae91aa-0c96-472e-ba8c-edd6162281f7
+
+controlPlane:
+  machineFlavor: vm.alaska.cpu.general.small
```

Comment:
Should we parameterise this value?

Reply:
think this would be too many vars kept in secrets to maintain, and no sensitive info is being exposed